src/main/app/case/models/task-list/index.ts

```diff
@@ -1,4 +1,4 @@
-export * from './enum/taskStatus';
+export * from './taskStatus';
 export * from './subTask';
 export * from './task';
 export * from './taskList';
```

Comment:
Instead of intellij and other editors doing it's thing without any intervention, I don't see why I would want to create these files in our project at this moment. I know the reason why they are there but not convinced.

Reply:
I think you are right @KiranVarma02  we can remove this file. The taskStatus and all can be directly imported in the other files as a direct import and I dont see an added advantage doing like this in the same project.
If it is a different project exporting a few classes in one file will makes sense.

Reply:
I also agree that this file can be removed, I prefer direct import.